build.gradle

```diff
@@ -75,7 +75,7 @@ ext {
     isMasterBranch = gitBranch == "master"
     isDevelopBranch = gitBranch == "develop"
     canUploadArchives = isMasterBranch || isDevelopBranch
-    isSnapshotVersion = project.version.contains("NEXT")
+    isSnapshotVersion = project.version.endsWith('-SNAPSHOT')
```

Comment:
Minor
```suggestion
    isSnapshotVersion = project.version.endsWith('NEXT-SNAPSHOT')
```

Reply:
This code has been finally removed. Publishing properties are managed in the `jenkins-library`.